app/views/provider_interface/hesa_export/new.html.erb

```diff
@@ -17,7 +17,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      <%= t('page_titles.provider.export_hesa_data') %>
+      <%= "Export data since #{EndOfCycleTimetable::CYCLE_DATES[RecruitmentCycle.current_year][:apply_reopens].to_s(:govuk_date)} for Higher Education Statistics Agency (HESA)" %>
```

Comment:
Maybe this can be moved to the locale file similarly to the other hesa export page title